vpr/src/route/route_common.h

```diff
@@ -22,7 +22,7 @@ void pathfinder_update_acc_cost_and_overuse_info(float acc_fac, OveruseInfo& ove
 
 float update_pres_fac(float new_pres_fac);
 
-t_trace* update_traceback(t_heap* hptr, ClusterNetId net_id);
+t_trace* update_traceback(t_heap* hptr, int target_pin, ClusterNetId net_id);
```

Comment:
target_pin -> target_net_pin_index 
and add a comment on usage.

Reply:
Done.